### ISSUE TITLE
System.NullReferenceException: Object reference not set to an instance of an object.   at System.Data.Entity.Core.Metadata.Edm.DataModelValidationRuleSet.<>c__DisplayClass1.<GetRules>b__0(DataModelValidationRule r)

### ISSUE THREAD

**Issue by jinek** [aspnet/EntityFramework6#635]
Describe what is not working as expected.

If you are seeing an exception, include the full exceptions details (message and stack trace).

```
Exception message: Object reference not set to an instance of an object
Stack trace: at System.Data.Entity.Core.Metadata.Edm.DataModelValidationRuleSet.<>c__DisplayClass1.<GetRules>b__0(DataModelValidationRule r)
  at System.Linq.Enumerable.WhereListIterator`1.MoveNext()
  at System.Data.Entity.Core.Metadata.Edm.EdmModelValidationVisitor.EvaluateItem(MetadataItem item)
  at System.Data.Entity.Core.Metadata.Edm.DataModelValidator.Validate(EdmModel model, Boolean validateSyntax)
  at System.Data.Entity.Core.Metadata.Edm.EdmModel.Validate()
  at System.Data.Entity.DbModelBuilder.Build(DbProviderManifest providerManifest, DbProviderInfo providerInfo)
  at System.Data.Entity.DbModelBuilder.Build(DbConnection providerConnection)
  at System.Data.Entity.Internal.LazyInternalContext.CreateModel(LazyInternalContext internalContext)
  at System.Data.Entity.Internal.RetryLazy`2.GetValue(TInput input)
  at System.Data.Entity.Internal.LazyInternalContext.InitializeContext()
  at System.Data.Entity.Internal.LazyInternalContext.get_ModelBeingInitialized()
  at System.Data.Entity.Infrastructure.EdmxWriter.WriteEdmx(DbContext context, XmlWriter writer)
  at System.Data.Entity.Utilities.DbContextExtensions.GetModel(Action`1 writeXml)
  at System.Data.Entity.Migrations.DbMigrator..ctor(DbMigrationsConfiguration configuration, DbContext usersContext, DatabaseExistenceState existenceState, Boolean calledByCreateDatabase)
  at System.Data.Entity.Migrations.DbMigrator..ctor(DbMigrationsConfiguration configuration)
```

### Steps to reproduce
Happened only once during startup.

### Further technical details
EF version: 6.2.0
Database Provider: EntityFramework.SqlServer
Operating system: Azure Web App
IDE:  JetBrains Rider 2018.2.3

**Comment by ajcvickers** [comment]
**EF Team Triage:** This issue is lacking enough information for us to be able to effectively triage it. In particular, it is missing the following information requested in the new issue template. Can you please provide this information?

> ### Steps to reproduce
Ideally include a complete code listing that we can run to reproduce the issue.
Alternatively, you can provide a project/solution that we can run.

BTW we're not just doing this to be mean :smile:... we get a lot traffic on this project and it takes time to attempt to reproduce an issue based on fragments of information. In addition, our attempt is often unsuccessful as the exact conditions required to hit the issue are often not explicitly included in the code provided. To ensure we maximize the time we have to work on fixing bugs, implementing new features, etc. we ask that folks give us a self-contained way to reproduce an issue.

For a guide on submitting good bug reports, read [Painless Bug Tracking](http://www.joelonsoftware.com/articles/fog0000000029.html). 

*BTW this is a canned response and may have info or details that do not directly apply to this particular issue. While we'd like to spend the time to uniquely address every incoming issue, we get a lot traffic on the EF projects and that is not practical. To ensure we maximize the time we have to work on fixing bugs, implementing new features, etc. we use canned responses for common triage decisions.*

**Comment by jinek** [comment]
Hello @ajcvickers !
Thank you for your answer, I have no information that you are asking for. This exception came from production.
For now I can only think we can add some code to distinguish all possibly null pointers in the method above.
Or we can just leave it unless someone will get same issue and will be able to reproduce it.

**Comment by ajcvickers** [comment]
**EF Team Triage:** Closing this issue as the requested additional details have not been provided and we have been unable to reproduce it.

*BTW this is a canned response and may have info or details that do not directly apply to this particular issue. While we'd like to spend the time to uniquely address every incoming issue, we get a lot traffic on the EF projects and that is not practical. To ensure we maximize the time we have to work on fixing bugs, implementing new features, etc. we use canned responses for common triage decisions.*
